### ISSUE TITLE
Devanagari Vedic Extensions $1CD0, $1CD1, $1CD2 cannot be attached or positioned with Numerals

### ISSUE THREAD

**Issue by advaita56** [MicrosoftDocs/typography-issues#691]
1. Bug3 in OPENTYPE Devanagari scripts “dev” or “dev2”
2. **Using Microsoft Word 2016, 2019, Office 365 on Windows 7 and Windows 10**
3. ### Renders fine Using Microsoft Word 2010 on Windows 7
4. Font Sanskrit2020.otf https://sourceforge.net/projects/advaita-sharada-font/files/Devanagari/
5. Devanagari Unicode Block $0900 to $097F
6. Vedic Extensions Unicode Block $1CD0 to $1CFF
a)	glyphs $1CD0, $1CD1, $1CD2 cannot be anchored or precisely positioned for numerals using GPOS / GSUB. 
b)	The geometric shape $25CC keeps getting displayed.
7. अं अँ कं कँ हं अः॑ कं॑ ह॑ Whereas with glyphs $0900 to $0939 these can be accurately positioned and the geometric shape $25CC is never in the way. 
[bug3 in OpenType Devanagari script.docx](https://github.com/MicrosoftDocs/typography-issues/files/5552691/bug3.in.OpenType.Devanagari.script.docx)
![bug3 in OpenType Devanagari script](https://user-images.githubusercontent.com/63571731/99376106-81331900-28ea-11eb-934e-106e16f80669.jpg)
[bug3 in OpenType Devanagari script.pdf](https://github.com/MicrosoftDocs/typography-issues/files/5552694/bug3.in.OpenType.Devanagari.script.pdf)

**Comment by alib-ms** [comment]
Thank you for the feedback, can you please share the sample text string shown in your picture.

**Comment by advaita56** [comment]
Yes sure. Please see attached word document. Vedic Extensions Unicode Block
$1CD0 to $1CDF.
You may refer to the font file Sanskrit2020 at
https://sourceforge.net/projects/advaita-sharada-font/files/Devanagari/

Jai Gurudev

**Comment by alib-ms** [comment]
Windows 10 includes "Sanskrit Text" font. Can you please let me know your observation using this font. Here is what is displayed on my end.
![image](https://user-images.githubusercontent.com/36778811/99554090-ad46ac80-2973-11eb-8f47-60ba7566fe41.png)

**Comment by tiroj** [comment]
I think I had not anticipated these marks being applied to numerals, so the Sanskrit Text font may need some additional anchors defined to correctly position these combinations.

**Comment by alib-ms** [comment]
Thank you John, agree we can improve the positioning, however, we are not seeing the dotted circle as the bug reported using this font on Word + Windows 10.

**Comment by advaita56** [comment]
Our end shows the geometric shape. The diacritics $1CD0, $1CD1, $1CD2 do
not attach to numerals at all.
Please see attached word file and image file.

**Comment by advaita56** [comment]
Please note that in "*google docs*" the diacritics get attached to the
Numerals, since google docs has no uniscribe shaping engine for Devanagari.

In *Microsoft Word* these do not get attached at all. The dotted circle
always shows up.
Also in Microsoft Word, when we use Insert Symbol, the characters sets
"Vedic Extensions" and "Devanagari Extended" are not displayed in the
header, nor are the unicode names for these characters displayed.

**Comment by alib-ms** [comment]
For the following string 
०᳐१᳑२᳒३᳓४᳔५᳕६᳖७᳗८᳘९᳟
OS:Windows 10 
Font:"Sanskrit Text"
Application: Microsoft Word
![image](https://user-images.githubusercontent.com/36778811/99707583-81015d80-2a51-11eb-94ef-9823e11bbec1.png)

Application: Notepad 
![image](https://user-images.githubusercontent.com/36778811/99707537-6c24ca00-2a51-11eb-95a2-9fdbac7d47b6.png)

I am curious what the above string looks like for you using Windows 10 and Word/Notepad with Sanskrit Text font.

**Comment by advaita56** [comment]
For the following string
०᳐१᳑२᳒३᳓४᳔५᳕६᳖७᳗८᳘९᳟
We also included the text क᳐ ख᳑ ग᳒ घ᳓अ᳢ इ᳕ उ᳖ ट᳗ ज᳘ च᳟ to show what happens
with Sanskrit vowels and consonants.

OS:Windows 10, Font:"Sanskrit Text" , Application: Microsoft Word
[image: Word Sanskrit Text.PNG]
 Application: Notepad

[image: Notepad Sanskrit Text.png]
Notepad is same as yours. But Word inserts the geometric shape for Numerals
with diacritics...
Thanks so much for looking into this.
